src/intervals/construction.jl

```diff
@@ -220,7 +220,7 @@ Create an equivalent `BigFloat` interval to a given `AbstractFloat` interval.
 """
 function bigequiv(a::Interval{T}) where {T <: AbstractFloat}
     lock(precision_lock) do
-        return setprecision(precision(T)) do  # precision of T
+        setprecision(precision(T)) do  # precision of T
             return Interval{BigFloat}(a)
```

Comment:
If we dont need the return statements above that we can drop this return statement, too.